Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/Util.cs

```diff
@@ -61,7 +61,8 @@ internal static bool TryConvertHexStringToHex(string hex)
                 arr[i] = Convert.ToByte(hex.Substring(i * 2, 2), 16);
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/26a0e5ba281238531e161fb600aa7493a3a50e5a/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/Util.cs#L2
You're not using this namespace, so it can be removed.